enviro/__init__.py

```diff
@@ -153,56 +153,105 @@ def stop_activity_led():
 print("    -  --  ---- -----=--==--===  hey enviro, let's go!  ===--==--=----- ----  --  -     ")
 print("")
 
+def reconnect_wifi(ssid, password, country):
+    import time
```

Comment:
I would suggest to match the 2 spaces used by the original code, because right now it's almost impossible to see what's actually different

Reply:
Indeed, that was an error. Adjusted.